homeassistant/components/google_drive/services.yaml

```
@@ -0,0 +1,24 @@
+append_sheet:
+  name: Append to Sheet
+  description: Append data to a worksheet in Google Sheets.
+  fields:
+    config_entry:
+      name: Config entry
+      description: Config entry to use.
```

```suggestion
      description: The sheet to add data to
```

(or no description as it's pretty self explanatory)